themes/falconf/layouts/index.html

```diff
@@ -4,6 +4,7 @@
   <div id="df-sidebar-wrapper">
     <nav class="df-sidebar-nav">
       <a href="#about">About</a>
+      <a href="#tickets">Tickets</a>
```

Comment:
Wrap it in hugo block like this, please, for consistency:

```
{{ if .Site.Params.sectionTickets.enabled }}
    <a href="#tickets">Tickets</a>
{{ end }}
```

Reply:
Done! But why other links are not inside such blocks? @skibish

Reply:
Because others are currently default. We will get to them soon